src/formatting/chains.rs

```diff
@@ -147,7 +147,7 @@ impl ChainItemKind {
 
     fn from_ast(context: &RewriteContext<'_>, expr: &ast::Expr) -> (ChainItemKind, Span) {
         let (kind, span) = match expr.kind {
-            ast::ExprKind::MethodCall(ref segment, ref expressions) => {
+            ast::ExprKind::MethodCall(ref segment, ref expressions, _) => {
```

Comment:
Cool, now we have a span for the function of a method call. I hope we could utilize this in rustfmt.